sdk/include/opentelemetry/sdk/metrics/aggregation/histogram_aggregation.h

```diff
@@ -108,6 +108,26 @@ void HistogramDiff(HistogramPointData &current, HistogramPointData &next, Histog
   diff.record_min_max_ = false;
 }
 
+template <class T>
+size_t BucketBinarySearch(T value, std::vector<double> &boundaries)
```

Comment:
Can we use std::binary_search here?

Reply:
“std::binary_search” will not provide index of element for insertion, it just provides true or false if element is found.